docs/UserGuide.md

```diff
@@ -45,27 +45,45 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
 **:information_source: Notes about the command format:**<br>
 
 * Words in `UPPER_CASE` are the parameters to be supplied by the user.<br>
-  e.g. in `add n/NAME`, `NAME` is a parameter which can be used as `add n/John Doe`.
+  e.g. in `ingredient NAME`, `NAME` is a parameter which can be used as `ingredient milk`.
 
 * Items in square brackets are optional.<br>
-  e.g `n/NAME [t/TAG]` can be used as `n/John Doe t/friend` or as `n/John Doe`.
```

Comment:
I think we should still use this prefix, to distinguish the input is either a name or a tag or something else.

Same applies to below, line 57

Reply:
I agree.

Reply:
thanks, resolved